utils/ipfs.py

```diff
@@ -48,6 +49,41 @@ class IPFSUploadResult(TypedDict):
     messages: list[Tuple[str, str]]
 
 
+# alternative for upload_str_to_web3_storage
+def _upload_str_to_infura_io(text: str) -> str:
+    text_bytes = text.encode("utf-8")
+    text_file = io.BytesIO(text_bytes)
+    files = {"file": text_file}
+    (projectId, projectSecret) = get_infura_io_keys()
+
+    endpoint = "https://ipfs.infura.io:5001"
+
+    response = requests.post(endpoint + "/api/v0/add?cid-version=1", files=files, auth=(projectId, projectSecret))
+    response.raise_for_status()
+    response_json = response.json()
+
+    return response_json.get("Hash")
+
+
+# alternative for upload_str_to_web3_storage
+def _upload_str_to_pinata_cloud(text: str) -> str:
+    text_bytes = text.encode("utf-8")
+    text_file = io.BytesIO(text_bytes)
+    files = {"file": text_file}
+    pinata_cloud_token = get_pinata_cloud_keys()
```

Comment:
looks like it should be `get_pinata_cloud_token ` instead of  `get_infura_io_keys`
the same thing on line 104